### DISCUSSION TITLE
小米门窗传感器状态为接触器/未接触

### DISCUSSION THREAD

**Discussion by dldui-captain** [XiaoMi/ha_xiaomi_home#449] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/e03373e9-4561-417e-911c-d03c18f5665e)

小米门窗传感器状态为接触器/未接触，如何调整 关闭/打开  ？

**Comment by pcr** [comment, marked as answer]
希望这个项目后面能直接把bool型的只读属性映射为binary-sensor，这样就彻底解决了。
现阶段只能用HA的辅助元素建一个模版实例，人工把这个转换成一个二位传感器。

**Reply by micturkey** [reply]
我提了一个PR，https://github.com/XiaoMi/ha_xiaomi_home/pull/236
miot auto的作者也提了个PR，https://github.com/XiaoMi/ha_xiaomi_home/pull/272
其实门窗和水浸的好解决，人在和人体的怎么映射到binary sensor才麻烦
